components/link_tooltip/link_tooltip.tsx

```diff
@@ -76,7 +90,7 @@ export default class LinkTooltip extends React.PureComponent {
         }, Constants.OVERLAY_TIME_DELAY_SMALL);
     };
 
-    render() {
+    public render(): JSX.Element {
```

Comment:
nit: I don't think we came up with consistent way of doing this but some (including me) voted for simply using inferred (less noise)
```suggestion
    public render() {
```

Reply:
I agree that it looks a lot cleaner, I'll amend asap.